### PR TITLE
Erc677 callback

### PR DESCRIPTION
**Pull request by isentropy**
- add custom tokenbridge home mediator to prebaked OpenEthereum image. New mediator calls mint+transferAndCall so that recipient contract can automatically update when tokens received
- update DU2 contracts to process transferAndCall

### CODE REVIEW

**Comment by isentropy** [comment]
added new NodeRegistry node per https://linear.app/streamr/issue/ETH-51/deploy-a-new-noderegistry

File: index.js [hunk 1]
```diff
@@ -198,6 +198,8 @@ async function smartContractInitialization() {
     initialUrls.push('{"ws": "ws://10.200.10.1:30302", "http": "http://10.200.10.1:30302"}')
     initialNodes.push('0xf2C195bE194a2C91e93Eacb1d6d55a00552a85E2')
     initialUrls.push('{"ws": "ws://10.200.10.1:30303", "http": "http://10.200.10.1:30303"}')
+    initialNodes.push('0xde1112f631486CfC759A50196853011528bC5FA0')
```
Comment (jtakalai) [inline index.js:201]: is this added node related to the ERC677 token change?

**Review by jtakalai — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (hpihkala) [reply index.js:201]: The goal was to deploy another `NodeRegistry`, this shouldn't be here at all, needs to be removed

Comment (hpihkala) [reply index.js:201]: The goal in ETH-51 was to deploy another `NodeRegistry`, this shouldn't be here at all, needs to be removed

**Review by hpihkala — CHANGES_REQUESTED** [review]

**Comment by isentropy** [comment]
@hpihkala I'll remove and make new PR with new NodeRegistry

**Comment by isentropy** [comment]
@hpihkala  removed node. Can I merge?
